aveas_openlabel/attributes/open_drive.py

```diff
@@ -37,6 +37,28 @@ class OpenDrive__RoadId(TextData):
     """Is always 'open_drive/road_id'"""
 
 
+@dataclass
+class OpenDrive__LaneId(TextData):
+    """Lane ID the object is moving on corresponding to the Road ID (stated in OpenDrive__RoadId) in the OpenDRIVE file"""
+
+    val: str = field(default_factory=lambda: no_default(field="OpenDrive__LaneId.val"), metadata=required)
+    """The OpenDRIVE lane ID"""
+
+    name: Literal["used_lane_id"] = field(default="used_lane_id")
```

Comment:
The value has to be `open_drive/lane_id`

Reply:
geändert